modules/container-definition/outputs.tf

```diff
@@ -0,0 +1,22 @@
+################################################################################
+# Container Definition
+################################################################################
+
+output "container_definition" {
+  description = "Container definition"
+  value       = local.container_definition
```

Comment:
Should this output be marked as sensitive? I see that in `modules/service/outputs.tf` it is marked so, so here it should probably be also marked?

Reply:
This was my bad - in the examples I was using an SSM parameter data source to pull a container image and the second you use that, it requires outputs to be marked as sensitive. So for now, I removed that and used a static string and will leave this up to users to mark the output as sensitive (if they need to)